check_source.py

```diff
@@ -72,6 +74,18 @@ def check_source_submission(self, source_project, source_package, source_revisio
                         'See https://en.opensuse.org/openSUSE:How_to_contribute_to_Factory#How_to_request_a_new_devel_project for details.'
                     ) % (source_project, target_project)
                     return False
+        else:
+            if source_project.endswith(':Update'):
+                # Allow for submission like:
+                # - source: openSUSE:Leap:15.0:Update/google-compute-engine.8258
+                # - target: openSUSE:Leap:15.1/google-compute-engine
+                # Note: home:jberry:Update would also be allowed via this condition,
```

Comment:
to avoid that one would have to check the project meta to check for maintenance_release:

  \<project name="openSUSE:Leap:15.0:Update" kind="maintenance_release">

Reply:
Can that not be done in home project?

I was considering using the prior project walking code, but since there exists the difference in names between IBS and OBS that seemed like a bad idea since this will likely end up being used on IBS as well. So left as is since not mission critical.

Reply:
the maintenance_release project kind is in both SLE and openSUSE. Can be used in home projects too, that is true